app/src/main/java/com/codepath/apps/critterfinder/models/SearchFilter.java

```diff
@@ -41,8 +41,23 @@ public String toString() {
         }
     }
 
+    public enum Animal {
+        DOG("dog"),
+        CAT("cat");
+
+        private final String mAnimal;
+
+        Animal(String animal) {
+            mAnimal = animal;
+        }
+        public String toString() {
+            return this.mAnimal;
+        }
+    }
```

Comment:
we'll have a small merge conflict depending on whose branch gets merged first as this class now has a Species type that has dog/cat.